components/support/ktx/src/test/java/mozilla/components/support/ktx/android/content/ContextTest.kt

```diff
@@ -6,29 +6,50 @@ package mozilla.components.support.ktx.android.content
 
 import android.app.ActivityManager
 import android.content.Context
+import android.view.accessibility.AccessibilityManager
 import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertNull
+import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith
+import org.mockito.Mockito.`when`
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.spy
 import org.robolectric.RobolectricTestRunner
 import org.robolectric.RuntimeEnvironment
+import org.robolectric.Shadows.shadowOf
+import org.robolectric.util.ReflectionHelpers
 
 @RunWith(RobolectricTestRunner::class)
 class ContextTest {
+
+    private var context: Context? = null
+    private var accessibilityManager: AccessibilityManager? = null
+
+    @Before
+    fun startUp() {
+        context = RuntimeEnvironment.application
+        accessibilityManager = context!!.getAccessibilityManager()
+    }
+
     @Test
     fun `systemService() returns same service as getSystemService()`() {
         val context = RuntimeEnvironment.application
 
         assertEquals(
-            context.getSystemService(Context.INPUT_METHOD_SERVICE),
-            context.systemService(Context.INPUT_METHOD_SERVICE))
+                context.getSystemService(Context.INPUT_METHOD_SERVICE),
+                context.systemService(Context.INPUT_METHOD_SERVICE))
 
         assertEquals(
-            context.getSystemService(Context.ACTIVITY_SERVICE),
-            context.systemService(Context.ACTIVITY_SERVICE))
+                context.getSystemService(Context.ACTIVITY_SERVICE),
+                context.systemService(Context.ACTIVITY_SERVICE))
 
         assertEquals(
-            context.getSystemService(Context.LOCATION_SERVICE),
-            context.systemService(Context.LOCATION_SERVICE))
+                context.getSystemService(Context.LOCATION_SERVICE),
+                context.systemService(Context.LOCATION_SERVICE))
```

Comment:
nit: All those whitespaces changes seem unnecessary?

Reply:
Oops. My IDE did that. I will correct it